akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -94,6 +94,10 @@ akka.cluster.sharding {
   # works only for state-store-mode = "ddata"
   updating-state-timeout = 5 s
 
+  # The interval beteween which remembered entities are restarted when initialing the shard
```

Comment:
@pbarron could you update the parsing of this property to handle `off`. We have similar in other places.